src/dns/nm-dns-manager.c

```diff
@@ -1637,28 +1637,25 @@ _resolvconf_resolved_managed (void)
 		"/run/systemd/resolve/resolv.conf",
 		"/lib/systemd/resolv.conf",
 		"/usr/lib/systemd/resolv.conf",
+		NULL
```

Comment:
let's not make the array NULL terminated, instead use a index variable like:

    for (i = 0; i < G_N_ELEMENTS (resolved_paths); i++) {
          // use resolved_paths[i]
    }